scripts/upload-compose-secrets.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+set -eu
+
+if [ "${AWS_ACCOUNT}" = "dev" ]
+   then
+        export PASSWORD_STORE_DIR="${HOME}"/.paas-pass
+        COMPOSE_ACCESS_TOKEN=$(pass "compose/integration_tests/access_token")
+   else
+        export COMPOSE_PASSWORD_STORE_DIR="${HOME}"/.paas-pass-high
+        COMPOSE_ACCESS_TOKEN=$(pass "compose/${AWS_ACCOUNT}/access_token")
```

Comment:
This should be using the integration_tests token in both environments. The CI api keys should only be used by the CI CF deployment (they belong to the CI PaaS-CF user).